Gordon360/Models/CCT/LostAndFound/MissingReports.cs

```diff
@@ -0,0 +1,78 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
```

Comment:
So this is for data entry users and MissingItemData is for general?

Reply:
MissingReports is the actual database table.  MissingItemData is actualy a SQL view based on this table, but it also pulls data from the CCT.ACCOUNTS, CCT.FacStaff, and CCT.Students, to get names, phone numbers, and email addresses.  MissingItemData is what actually get's used in most places.  It's possible we don't need a C# model for all of these tables, so I might remove some of the models, since none of the code has to access tables like Statuses and Categories.